src/utils/updateAllDiscordUserRoles.ts

```diff
@@ -15,27 +16,34 @@ export async function updateAllDiscordUserRoles(this: LunarAssistant) {
   console.log("Running updateAllDiscordUserRoles cronjob!");
 
   // loop over every user
-  const usersSnapshot = await this.db.collection("users").get();
+
+  const users = (
+    await this.db.collection("root").doc("users").get()
+  ).data() as Users;
   const guildConfigsSnapshot = await this.db.collection("guildConfigs").get();
 
-  const numUsers = usersSnapshot.docs.length;
+  const numUsers = users.discordIds.length;
 
-  await usersSnapshot.docs.reduce(
-    (p, userDoc, index) =>
+  await users.discordIds.reduce(
```

Comment:
Same comment as earlier - this is sequential, which will affect performance. Not sure if this is the job run daily?

Reply:
This is intentional to not hit the public nodes too hard